Receipts/ReceiptTableViewManager.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+class ReceiptTableViewManager: NSObject, UITableViewDataSource {
+    
+    let tableView: UITableView
+    let receiptStore: ReceiptStore
+    var receipts: [Receipt] {
+        return receiptStore.receipts
+    }
+    
+    /// Initializes a new Receipt TableView Manager.
+    ///
+    /// - Parameter tableView: the tableView which wants it's dataSource to be this manager instance.
+    init(tableView: UITableView, receiptStore: ReceiptStore) {
+        self.tableView = tableView
+        self.receiptStore = receiptStore
+        super.init()
+        configureTableView()
+    }
+    
+    
+    /// This method will deselect any current table view selections while making sure to reload the previously selected cells as well.
+    /// This is a particularly helpful thing when coming back to a tableView after a user had selected a row for possible editing.
+    func deselectAndReloadCurrentTableViewSelection() {
+        if let indexPath = tableView.indexPathForSelectedRow {
+            tableView.deselectRow(at: indexPath, animated: false)
+            tableView.reloadRows(at: [indexPath], with: .automatic)
+        }
+    }
+    
+    //MARK: - UITableViewDataSource
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return receipts.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "ReceiptTableViewCell", for: indexPath) as! ReceiptTableViewCell
```

Comment:
Notice we just transferred the previous code. It will be it's own refactor. Keep your goals focused and small.